coral/coral/media/js/views/components/plugins/owner-occupier-report-workflow.js

```diff
@@ -0,0 +1,431 @@
+define([
+  'knockout',
+  'arches',
+  'viewmodels/editable-workflow',
+  'templates/views/components/plugins/owner-occupier-report-workflow.htm',
+  'views/components/workflows/licensing-workflow/widget-labeller',
+  'views/components/workflows/owner-occupier-report-workflow/owner-occupier-report-init-step'
+], function (ko, arches, EditableWorkflow, licensingWorkflowTemplate) {
```

Comment:
"licensingWorkflowTemplate" should probably be renamed.
Not major change though